model/font_simple.go

```diff
@@ -379,16 +381,27 @@ func (font *pdfFontSimple) ToPdfObject() core.PdfObject {
 	return font.container
 }
 
-// NewPdfFontFromTTFFile loads a TTF font and returns a PdfFont type that can be used in text
-// styling functions.
-// Uses a WinAnsiTextEncoder and loads only character codes 32-255.
+// NewPdfFontFromTTFFile is a wrapper around NewPdfFontFromTTF.
```

Comment:
Could you add a more descriptive function comment here, similar to the one of `NewPdfFontFromTTF`?